src/index.html

```diff
@@ -14,7 +14,7 @@ <h1>Today's To Do</h1>
           </div>
           <div class="input_and_add">
               <form action="">
-                  <input type="text" class="input-task" id="input-task" placeholder="Add to your list...">
+                  <input type="text" class="input" id="input-task" placeholder="Add to your list...">
```

Comment:
> Add a new JavaScript file and import it as a module:
-  it will contain methods related to the status updates (completed: true / false).

- [ ] You did great so far, :+1: But here are a few requirements you missed and they need to be implemented before moving to the next milestone. :+1: 
- [ ] According to the project requirement, the functionality to change the task to completed should be in a new file inside the module folder. Currently, you have the function in your index.js file, I'll suggest you follow the project requirement and update this. 
- [ ] As suggested by the previous reviewer about the add and remove task functionality, You did great including the `add` task function, Your todo app doesn't have the remove task which should have been implemented in the previous milestone. I'll suggest you implement that. Also, your UI doesn't have the delete icon for individual tasks. I'll suggest you add that too. :+1:  

- - Screenshot-{NO DELETE TASK BUTTON}:
![Screenshot from 2023-06-15 15-13-41](https://github.com/kkmanuu/To-Do-Lists/assets/107632780/d469cdbb-327d-4ca7-b9ba-8c91c54440c1)


- [ ] While trying to run your app, I noticed you have a couple of dependencies missing in your `package.json` file. Your project is missing ` webpack-cli, webpack-dev-server and css-loader`, I' 'll suggest you install these dependencies as they are needed for your project to run properly. :wave:

Reply:
@V-Blaze V-Blaze Done making changes